index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
Does the `basePath` configuration variable alleviate needing to update the readme?